acorn/src/options.js

```diff
@@ -91,14 +91,25 @@ export const defaultOptions = {
 
 // Interpret and default an options object
 
+let warnedAboutEcmaVersion = false
+
 export function getOptions(opts) {
   let options = {}
 
   for (let opt in defaultOptions)
     options[opt] = opts && has(opts, opt) ? opts[opt] : defaultOptions[opt]
 
-  if (options.ecmaVersion >= 2015)
+  if (options.ecmaVersion === "latest") {
+    options.ecmaVersion = 1e8
+  } else if (options.ecmaVersion == null) {
```

Comment:
If this is expected to be the rare "error case", let's move it to the end as less likely branch?

Reply:
I put it here to avoid relying on the not very obvious behavior of comparing to null/undefined in the next branch.

Reply:
I see. I guess in my mind it's "obviously" the same as zero by now, but I can see that argument as it's context-dependant :)

Reply:
You could use `else if (typeof options.ecmaVersion === 'number' && options.ecmaVersion >= 2015)` for the second condition and the 3rd could simply be `else`.

These are all small enough that it doesn't really matter 🙂

Reply:
You don't really need it, because `null` would behave correctly there too, but yeah, it doesn't matter much.